us-west1/config/gke.tf.no

```diff
@@ -0,0 +1,15 @@
+# Module for GKE cluster with separate node pool
+
+module "gke_cluster_test"{
+    source = "../../modules/gke"
+
+    name                        = "test-cluster"
+    location                    = "us-west1"
+
+    remove_default_node_pool    = true
```

Comment:
per my other comment, if you default this to true, you can remove this input.  since the module is set up to add a custom node pool, I cannot think of why we would want to not default this to true.  open to suggestions though.

Reply:
Yeah, I'll edit this so that the default is true and we don't have to pass this argument in the gke.tf file.